summerofcode/2023.md

```diff
@@ -24,3 +24,52 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ---
 
+## Ideas
+
+- [Knative](#knative)
+  - [Multiple Knative Eventing control planes](#multiple-knative-eventing-control-planes)
+  - [Eventing Sender Identity](#eventing-sender-identity)
+  - [NetworkPolicy support for Knative Serving](#networkpolicy-support-for-knative-serving)
+  - [Improving Observability in Knative Eventing](#improving-observability-in-knative-eventing)
```

Comment:
I've added this TOC as well. Let me know if you would like me to remove it.

Reply:
I appreciate the TOC. Usually i add it when we move projects from ideas to accepted. If folks update it as they add ideas i'm happy to leave it in, but if folks don't then I'd rather take it out and add it all at once with a tool at the end.